docs/src/modules/spring/pages/event-sourced-entities.adoc

```diff
@@ -123,6 +123,27 @@ IMPORTANT: For simplicity purposes, we are returning the internal state directly
 
 include::java:partial$eventsourced-snapshots.adoc[]
 
+== Deleting Entity
+
+Normally, Event Sourced Entities are not deleted because the history of the events typically provide business value.
+For certain use cases or for regulatory reasons the entity can be deleted.
+
+[source,java,indent=0]
+.src/main/java/com/example/shoppingcart/ShoppingCartEntity.java
+----
+include::example$spring-eventsourced-shopping-cart/src/main/java/com/example/shoppingcart/ShoppingCartEntity.java[tag=checkout]
+----
+<1> Emit final event before deletion, which is handled as any other event.
+<2> Instruction to delete the entity.
+
+When you give the instruction to delete the entity it will still exist from some time, including its events and snapshots. The actual removal of events and snapshots will be deleted later to give downstream consumers time to process all prior events, including the final event that was emitted together with the `deleteEntity` effect. By default, the existence of the entity is completely cleaned up after a week.
```

Comment:
```suggestion
When you give the instruction to delete the entity it will still exist for some time, including its events and snapshots. The actual removal of events and snapshots will be deleted later to give downstream consumers time to process all prior events, including the final event that was emitted together with the `deleteEntity` effect. By default, the existence of the entity is completely cleaned up after a week.
```